### PR TITLE
add group chat

### PR DESCRIPTION
**Pull request by coyoty**
add group chat if the documents app creates the id and the password for a document

### CODE REVIEW

File: js/ojsxc.js [hunk 1]
```diff
@@ -44,6 +44,23 @@ function onRosterToggle(event, state, duration) {
    }
 }
 
+function checkForChatRoom() {
+   if($("#document-title").attr("data-chatroom-password") && $("#document-title").attr("data-chatroom-password")!="undefined") {
```
Comment (sualko) [inline js/ojsxc.js]: Remove `!="undefined"` or use `typeof`.

File: js/ojsxc.js [hunk 2]
```diff
@@ -44,6 +44,23 @@ function onRosterToggle(event, state, duration) {
...
+      var roomId = $("#document-title").data("chatroom-name");
+      roomId = "jsxcDocumentChat"+roomId;
+      var roomName = $("#document-title").text();
+      var conferenceServer = jsxc.options.get('muc').server;
+      var nickname = Strophe.getNodeFromJid(jsxc.muc.conn.jid);
+      $("#chatRoomJoinContainer").html('<button id="joinChatroom">Join document chat</button>');
+      $("#joinChatroom").click(function(){
+         jsxc.storage.setUserItem('member', roomId, {});
+         jsxc.muc.join(roomId+"@"+conferenceServer, nickname, roomPassword, roomName, '', false, false);
+         jsxc.gui.window.open(roomId+"@"+conferenceServer);
```
Comment (sualko) [inline js/ojsxc.js]: I think it could be annoying, if a new chat window opens for every document.

**Review by sualko — CHANGES_REQUESTED** [review]
>  if the documents app creates the id and the password for a document

Is there a pending pr? Can you reference it?

File: js/ojsxc.js (hunk 2 above)
Comment (coyoty) [inline js/ojsxc.js]: this is the pull request: https://github.com/owncloud/documents/pull/665

**Comment by coyoty** [comment]
no longer using owncloud

**Pull request closed without merging**